mlrun/config.py

```diff
@@ -288,6 +288,12 @@
         "state": "online",
         "retry_api_call_on_exception": "enabled",
         "http_connection_timeout_keep_alive": 11,
+        # http client used by httpdb
+        "http": {
+            # when True, the client will verify the server's TLS
+            # sset to False for backwards compatibility.
+            "verify": False,
```

Comment:
agree with Saar we might want to split this to nuclio, httpdb, etc.
I think it should be configured for each client and also have an option to always verify.
maybe something like:
```
# list of clients to verify TLS, options are nuclio, httpdb, iguazio or "*" to verify for all.
"verify": []
```